core/src/test/java/org/apache/iceberg/TestMergeAppend.java

```diff
@@ -648,4 +649,98 @@ public void testInvalidAppendManifest() throws IOException {
             .appendManifest(manifestWithDeletedFiles)
             .commit());
   }
+
```

Comment:
Nit: unnecessary newline.

Reply:
Thanks. Will remove it in https://github.com/apache/incubator-iceberg/pull/922.